enatega-multivendor-admin/lib/utils/methods/timeAgo.ts

```
@@ -0,0 +1,25 @@
+export const timeAgo = (timestamp: number): string => {
+  if (isNaN(timestamp) || timestamp <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)